opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -448,6 +459,107 @@ message HistogramDataPoint {
   uint32 flags = 10;
 }
 
+// ExponentialHistogramDataPoint is a single data point in a timeseries that describes the
+// time-varying values of a ExponentialHistogram of double values. A ExponentialHistogram contains
+// summary statistics for a population of values, it may optionally contain the
+// distribution of those values across a set of buckets.
+//
+message ExponentialHistogramDataPoint {
+  // The set of key/value pairs that uniquely identify the timeseries from
+  // where this point belongs. The list may be empty (may contain 0 elements).
+  repeated opentelemetry.proto.common.v1.KeyValue attributes = 1;
+
+  // StartTimeUnixNano is optional but strongly encouraged, see the
+  // the detiled comments above Metric.
```

Comment:
```suggestion
  // the detailed comments above Metric.
```

Reply:
This had been copied a bunch of places, now all fixed!